test/vendor/knative.dev/serving-operator/pkg/client/clientset/versioned/clientset.go

```diff
@@ -22,32 +22,24 @@ import (
 	discovery "k8s.io/client-go/discovery"
 	rest "k8s.io/client-go/rest"
 	flowcontrol "k8s.io/client-go/util/flowcontrol"
-	servingv1alpha1 "knative.dev/serving-operator/pkg/client/clientset/versioned/typed/serving/v1alpha1"
+	operatorv1alpha1 "knative.dev/serving-operator/pkg/client/clientset/versioned/typed/serving/v1alpha1"
```

Comment:
Interesting. Is this coming after changing dependency versions and calling `go dep update` Or are we changing this only in our fork in the vendor dir.

Reply:
There is no post-editing of any vendor files. This is strictly pulled in from `dep ensure`